manifest/manifestlist/manifestlist.go

```diff
@@ -0,0 +1,134 @@
+package manifestlist
```

Comment:
Might be good to have a package doc for the concept of manifest lists with a few examples of how they might be used. Let's not hold up the PR on this, but it might be helpful.